src/ComicsPage.module.css

```diff
@@ -1,5 +1,5 @@
 *{
-	margin: 0;
+    margin: 0;
 	padding: 0;
```

Comment:
padding also should have 2 spacing tab))

Reply:
and rename files please, there is a convention to name files from lowercase letter except react component files